app/src/main/java/com/rahobbs/todo/database/TodoSchema.java

```diff
@@ -0,0 +1,21 @@
+package com.rahobbs.todo.database;
+
+/**
+ * Created by rachael on 2/9/16.
+ * Defines database for to-do items
+ */
+public class TodoSchema {
+    public static final class TodoTable{
```

Comment:
nested classes! If there is a reason for them fine, but generally in Java I'd try and avoid them.

Also unless you think for clarity or organization it makes sense to have the scheme in it's own class, I would maybe move it into the database class or with the object it is mapping. that way you don't have random schema classes floating around and it's easy to find things when you're looking for them.